include/flatbuffers/idl.h

```diff
@@ -578,6 +578,7 @@ struct IDLOptions {
   bool strict_json;
   bool output_default_scalars_in_json;
   int indent_step;
+  bool cpp_minify_enums{};
```

Comment:
Odd to have `bool` with {}

either leave uninitialized, or default to `false`

Reply:
default is anyways false and `{}` initialises it to false anyways.